lisa/features/nvme.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import re
+from typing import Any, List
+
+from lisa.feature import Feature
+from lisa.tools import Lspci, Nvmecli
+from lisa.tools.lspci import PciDevice
+from lisa.util.logger import get_logger
+
+FEATURE_NAME_NVME = "Nvme"
```

Comment:
The name doesn't need to follow programing name convention, and it should follow custom usage. So it should be "NVME"